src/Cart.java

```diff
@@ -17,6 +18,21 @@ public void addProductInCart(Product product) {
         devices.add(product);
     }
 
+    //METODO CHE AGGIUNGE AL CARRELLO TRAMITE ID
+    public void addToCartById(int id, Cart cart) {
```

Comment:
Non prendi i ningresso int id e Cart cart, ma prendi in ingresso: ProdcutFilter productFilter e Warehouse warehouse.